lessons/03-express-templates/inClassExercises.md

```diff
@@ -0,0 +1,12 @@
+##In-class Exercises
+1. Finish the tutorial in the [readme](https://github.com/olinjs/olinjs/tree/master/lessons/03-express-templates-mongo) and create a webapp that uses express and handlebars to display a list of classes at Olin in your web browser. Build off of the hello world application you did for homework. Once you have completed this, show the teaching team your webapp in a browser.
```

Comment:
- web app
- Express
- Handlebars
- Perhaps organize a list of *your* classes, their professors and locations, (rather than a list of all classes at Olin), to keep it shorter, but add a bit more structured information.